crates/loam-cli/examples/soroban/nft/src/nft.rs

```diff
@@ -0,0 +1,126 @@
+use loam_sdk::{
+    soroban_sdk::{self, contracttype, env, Address, Bytes, Lazy, Map, Vec},
+    IntoKey,
+};
+use loam_subcontract_core::Core;
+
+use crate::{
+    subcontract::{IsInitable, IsNonFungible},
+    Contract,
+};
+
+#[contracttype]
+#[derive(IntoKey)]
+pub struct MyNonFungibleToken {
+    admin: Address,
+    name: Bytes,
+    total_count: u32,
```

Comment:
Why are we making the upper limit so small?? Surely everyone alive will want one of our NFTs! 🙃

Reply:
Lol, you're right! Do you think `u128` is too ambitious? 😜

Reply:
Plenty ambitious! It's 10^14 times as many stars [probably](https://www.space.com/26078-how-many-stars-are-there.html) exist in the observable universe. If every one had 10 planets or off-planet-habs and each of those had ten billion people who wanted one of our NFTs, we'd still have 1000x more address space than we needed.